### ISSUE TITLE
How to compile on OPI PC

### ISSUE THREAD

**Issue by ywandy** [avafinger/gc2035#3]
My OPI PC have linux header files in /usr/src,how can i compile this driver on my OPI?

**Comment by avafinger** [comment]
I don't think you can compile the driver outside of linux tree. All the needed headers have relative path to kernel (no always linux headers), so you need to fix this. You better build the driver inside tree.

**Comment by ril3y** [comment]
I am getting this error when trying to compile:

/home/ril3y/gc2035/gc2035.c:13:35: fatal error: media/v4l2-chip-ident.h: No such file or directory
 #include <media/v4l2-chip-ident.h>
                                   ^
compilation terminated.
scripts/Makefile.build:334: recipe for target '/home/ril3y/gc2035/gc2035.o' failed
make[2]: *** [/home/ril3y/gc2035/gc2035.o] Error 1
Makefile:1522: recipe for target '_module_/home/ril3y/gc2035' failed
make[1]: *** [_module_/home/ril3y/gc2035] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-4.14.52-sunxi64'
Makefile:4: recipe for target 'default' failed
make: *** [default] Error 2

where is this file located at? v4l2-chip-ident.h

Is this deprecated?

**Comment by avafinger** [comment]
This driver is for kernel 3.x.
